### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by visorokin**
This xml file is custom console extension file. Once placed in a folder ..\AdminConsole\XmlStorage\Extensions\Actions\172d85e7-bb7a-4479-a6a2-768f175b75cb, it will add additional context menu item to task sequence deployment objects in Configuration Manager Console. The menu item looks similar to existing menu item with the name "Show Status Messages" and same icon. It produces additional context menu item at the bottom  on a task sequence deployment object that launches status message viewer which shows ALL status messages for the selected task sequence deployment. New menu item works differently from the previously existing menu item. Previously existing item shows status message only for current site, site code and that is prompting user to enter specific  time period.

### CODE REVIEW

File: AllStatusMessagesForTsDeployment.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<ActionDescription Class="Executable" DisplayName="ShowStatusMessages" MnemonicDisplayName="MnemonicShowStatusMessages"
```
Comment (silveyy) [inline AllStatusMessagesForTsDeployment.xml:1]: I'd prefer not to put this in the root of the repository. How about src/Console/Extensions/Actions/ ?

Comment (visorokin) [reply AllStatusMessagesForTsDeployment.xml:1]: How does it work? Can I change location in this pull request or need somehow to abandon it and try again?

**Comment by visorokin** [comment]
Need to put ne file at src/Console/Extensions/Actions/

File: AllStatusMessagesForTsDeployment.xml (hunk 1 above)
Comment (silveyy) [reply AllStatusMessagesForTsDeployment.xml:1]: clone your branch on your local machine (vscode makes this very simple)
use git mv "path to source file" "path to destination file"
git commit -m "moving file"
git push

**Pull request closed without merging**